go.mod

```diff
@@ -5,8 +5,8 @@ go 1.17
 require (
 	github.com/compose-spec/compose-go v1.2.4
 	github.com/containerd/console v1.0.3
-	github.com/containerd/containerd v1.6.3-0.20220401172941-5ff8fce1fcc6
```

Comment:
not sure this change is needed in this PR

Reply:
buildkit master has updated to containerd v1.6.4, I think that's why this gets bumped? https://github.com/moby/buildkit/blob/master/go.mod#L11